src/adapter/graphql/src/queries/datasets/dataset_metadata.rs

```diff
@@ -76,17 +76,22 @@ impl DatasetMetadata {
         ctx: &Context<'_>,
         format: Option<DataSchemaFormat>,
     ) -> Result<Option<DataSchema>> {
-        let format = format.unwrap_or(DataSchemaFormat::Parquet);
-
-        let query_svc = from_catalog::<dyn domain::QueryService>(ctx).unwrap();
-        let res_schema = query_svc
-            .get_schema(&self.dataset_handle.as_local_ref())
-            .await
-            .int_err()?;
-
-        match res_schema {
-            Some(schema) => Ok(Some(DataSchema::from_parquet_schema(&schema, format)?)),
-            None => Ok(None),
+        match format {
+            Some(DataSchemaFormat::ArrowJson) => {
+                let query_svc = from_catalog::<dyn domain::QueryService>(ctx).unwrap();
+                let schema_ref_opt = query_svc
+                    .get_schema(&self.dataset_handle.as_local_ref())
+                    .await
+                    .int_err()?;
+
+                match schema_ref_opt {
+                    Some(schema_ref) => Ok(Option::from(DataSchema::from_arrow_schema(
+                        schema_ref.as_ref(),
+                    ))),
+                    None => Ok(None),
+                }
+            }
+            _ => unimplemented!(),
```

Comment:
What about all the other formats? You have tested all `DataSchema::from_*` constructors, but are not using them?

Reply:
I thought that in current_schema we are migrating to use only ArrowJson...

Reply:
should we use all 3:
 Parquet,
    ParquetJson,
    ArrowJson?

Reply:
changed

Reply:
all: Parquet, Parquet-Json, Arrow-Json are available